src/components/TheNavBar.vue

```diff
@@ -55,19 +53,25 @@
             </a>
 
             <div class="navbar-dropdown is-right">
-              <a :href="profilePageUrl" target="_blank" class="navbar-item">
+              <a
+                :href="profilePageUrl"
+                target="_blank"
+                class="navbar-item nav-item"
```

Comment:
hum, this doesn't seem right. We have `navbar-item` and `nav-item` ? Yet another class doesn't seem to me the right way to go forward as it adds to the complexity of the app.
It seems that we don't need that. And that it is also the occasion to clean the existing inconsistencies already there.
I'll do a proposition here.